app/config/events.php

```diff
@@ -189,7 +189,10 @@
             '$description' => 'This event triggers when a bucket is deleted.',
```

Comment:
Looks like a couple of the other team events are using `bucket` instead of `team` in the description, might as well fix them too